manubot/cite/curie/__init__.py

```diff
@@ -129,64 +134,33 @@ def _download_bioregistry() -> None:
             *(x.lower() for x in resource.pop("synonyms", [])),
         }
         # remove invalid prefixes as per https://github.com/manubot/manubot/pull/306#discussion_r744125504
-        resource["all_prefixes"] = sorted(
+        resource["synonyms"] = sorted(
             filter(valid_prefix_pattern.fullmatch, all_prefixes)
         )
         registry.append(resource)
     json_text = json.dumps(registry, indent=2, ensure_ascii=False)
     bioregistry_path.write_text(json_text + "\n", encoding="utf-8")
```

Comment:
Noting that we're still downloading the bioregistry from `https://github.com/biopragmatics/bioregistry/raw/main/exports/registry/registry.json` (a hardcoded URL) rather than using the bioregistry package to retrieve it. This PR preserves the design where a specific version of this package will be hardcoded to a specific data version of the bioregistry. But presumably the bioregistry has the ability to download the bioregistry dataset or perhaps comes with a version of the dataset?

Reply:
Yes it's also possible to completely switch, but doing it this way lets you decide when you want to update the bioregistry and potentially do your own overrides